test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -72,6 +86,15 @@ class HttpFilterTest : public testing::Test {
     request_headers_.setMethod("POST");
   }
 
+  void TearDown() override {
+    for (auto* t : timers_) {
+      // This will fail if any timer is un-disabled at the end of the test run.
```

Comment:
`is still enabled`? avoids double negative

Reply:
Yep, simplifying this one.